src/VirtualMonoRepo/source-mappings.json

```
@@ -144,7 +144,9 @@
                 "src/humanizer/samples/**/*.js",
                 "src/application-insights/**/*.exe",
                 "src/application-insights/**/*.dll",
-                "src/application-insights/**/*.zip"
+                "src/application-insights/**/*.zip",
+                "src/application-insights/**/NuGet.config",
+                "src/newtonsoft-json/Src/NuGet.Config"
```

Would it make sense to just make this a blanket exclusion for all repos?

```suggestion
                "src/**/NuGet.config"
```

---

@mthalman unfortunately submodules need to have their own set of rules each explicitly like this so that what we could safely separate what our repos have globally and what a 3rd party repository needs.

---

But we don't have anything in the `src` directory except for the external repos.

---

Yeah, but the tooling that does the synchronization will not apply any rules from the outside to submodules - it will only apply rules that are specifically under the path of the submodule

---

Ah, got it. Thanks